lib/member_row.rb

```diff
@@ -32,6 +36,22 @@ class MemberRow < Scraped::HTML
 
   private
 
+  def wanted_prefixes
+    %w(Prof.)
+  end
+
+  def unwanted_prefixes
+    %w(Hon.)
+  end
+
+  def prefixes
+    wanted_prefixes + unwanted_prefixes
+  end
+
+  def name_parts
+    noko[1].text.strip.split(' ')
```

Comment:
we generally use `.tidy` rather than `.strip`, as it takes care of more edge cases